[EDU-4820] feat: add new deploy flags (#1038)
aziontech/docs: src/content/docs/en/pages/devtools/cli/azion-cli/commands/deploy/deploy.mdx

The `--help` option displays more information about the `deploy` command.

#### auto

The `--auto` option runs the entire flow of the command without interruptions.

#### config-dir

> The `--config-dir` option specifies the relative path to where your custom `azion.json` and `args.json` files are stored (default is "azion").

Reviewer: ```suggestion
The `--config-dir` option specifies the relative path to where your custom `azion.json` and `args.json` files are stored. The default is "azion".
```

#### no-prompt

> The `--no-prompt` option, when sent, ensures that whenever the CLI would display an interactive prompt due to an error, it instead just returns the error.

Reviewer: ```suggestion
When sent, the `--no-prompt` option ensures that whenever the CLI displays an interactive prompt due to an error, it instead just returns the error.
```

#### skip-build

> The `--skip-build` option, when sent, prevents the build command from being called during the deploy process.

Reviewer: ```suggestion
When sent, the `--skip-build` option prevents the build command from being called during the deploy process.
```
